scripts/create-azure-file-share.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+RESOURCE_GROUP_NAME=$1
+STORAGE_ACCOUNT_NAME=$2
+
+# Create an azure file share for NeuVector persistent storage
+CONNECTION_STRING=$(az storage account show-connection-string -n ${STORAGE_ACCOUNT_NAME} -g ${RESOURCE_GROUP_NAME} --query 'connectionString' -o tsv)
```

Comment:
does the connection string need to be stored anywhere? like key vault?

Reply:
no - it's not really useful for anything but this

Reply:
i mean for us to be able to get for neuvector?
We won't have direct access to the subscription so won't be able to retrieve it if it isn't stored somewhere

Reply:
ok will upload to KV - we will need the account key to create the sealed secret for NV